configure.in

```diff
@@ -50,13 +50,13 @@ fi
 # check we're building against a supported version of PostgreSQL
 citusac_pg_config_version=$($PG_CONFIG --version 2>/dev/null)
 version_num=$(echo "$citusac_pg_config_version"|
-              $SED -e 's/^PostgreSQL \([[0-9]]*\)\.\([[0-9]]*\)\([[a-zA-Z0-9.]]*\)$/\1.\2/')
+              $SED -e 's/^PostgreSQL \([[0-9]]*\)\(\.[[0-9]]*\)\{0,1\}\(.*\)$/\1\2/')
```

Comment:
No, that doesn't solve the issue, because then we'll have to adjust the expect lines for every following patch release (which are released every 2-3 months).  Remember, 10 dropped one number from the numbering schemes.  It used to be that 9.5 -> 9.6 were "major" (feature adding) releases, and 9.5.0 -> 9.5.1 were patch/minor releases (bugfix only).  There's now one digit less. I.e. 10->11 will be a major release, and 10.0 -> 10.1 -> 10.2 will be minor releases (bugfix only).

I don't think it makes sense to address that in the test block below.